app/meldingen/incident/containers/IncidentContainer/actions.js

```diff
@@ -5,11 +5,60 @@
  */
 
 import {
-  DEFAULT_ACTION,
+  SET_INCIDENT,
+  CREATE_INCIDENT,
+  CREATE_INCIDENT_SUCCESS,
+  CREATE_INCIDENT_ERROR,
+  GET_CLASSIFICATION,
+  GET_CLASSIFICATION_SUCCESS,
+  GET_CLASSIFICATION_ERROR
 } from './constants';
 
-export function defaultAction() {
+export function setIncident(incident) {
```

Comment:
`export function setIncident()` or `export const setIncident = () =>`?
Same goes every export in the app, we have to keep this uniform